From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: matyalatte <matyalatte@gmail.com>
Date: Fri, 18 Apr 2025 00:17:52 +0000
Subject: [PATCH] Remove an include of err.h

---
 src/dispatch_common.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/dispatch_common.c b/src/dispatch_common.c
index 9f3dac4..0095b4f 100644
--- a/src/dispatch_common.c
+++ b/src/dispatch_common.c
@@ -164,7 +164,6 @@
 #include <windows.h>
 #else
 #include <dlfcn.h>
-#include <err.h>
 #include <pthread.h>
 #endif
 #include <string.h>
